src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_CommitUniqueCompletionListItem.cs

```diff
@@ -49,10 +49,10 @@ CommandState ICommandHandler<CommitUniqueCompletionListItemCommandArgs>.GetComma
             // we do want it through, it would be easy to get again simply by asking the model
             // computation to remove all filtering.
 
-            if (model.IsUnique)
+            if (model.IsUnique && model.SelectedItemOpt != null)
```

Comment:
we should have an assert in the model that certain values can't be true if SelectedItemOpt is null.